packages/eslint-config-addin/tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Builds failed in CI pipeline with error "error TS2688: Cannot find type definition file for 'node'.  May need to to add @types/node to package.json